osu.Game/Screens/Play/Player.cs

```diff
@@ -581,7 +581,7 @@ public void Restart()
         private ScheduledDelegate completionProgressDelegate;
         private Task<ScoreInfo> prepareScoreForDisplayTask;
 
-        private void updateCompletionState(ValueChangedEvent<bool> completionState)
+        private void updateCompletionState(ValueChangedEvent<bool> completionState, bool skipStoryboardOutro = false)
```

Comment:
`completionState` should no longer exist now, use `ScoreProcessor.HasCompleted.Value` instead